common/src/main/webpack/dev.webpack.config.js

```diff
@@ -39,6 +39,12 @@ const Web = Merge(
       contentBase: [__dirname, parts.rootDir],
       historyApiFallback: true,
       proxy: {
+        /*"/api/conditions": {
```

Comment:
I really like that we can proxy the backends this way

Reply:
Yes, it's nice since we can use the same `/api` endpoints as in Heroku, which uses its own proxy.

However, I'm not sure if we will be using this moving forward since it seems we are going to use mainly WebSockets, which aren't subject to CORS and therefore lifts the requirements for a proxy.

It might make sense to just use environment variables for the WebSocket addresses, with defaults to development values.